man/mono.1

```diff
@@ -1652,11 +1652,11 @@ Automatically generates sequence points where the
 IL stack is empty.  These are places where the debugger can set a
 breakpoint.
 .TP
-\fBgen-compact-seq-points\fR
```

Comment:
yeah, we cannot simple drop it without notice especially when it was documented. Put it back and make it do nothing with a warning of deprecation